core/tests/test_jmespath_expression.py

```diff
@@ -215,3 +215,8 @@ def test_jmespath_in():
 
     assert expression.search({"el": 1})
     assert not expression.search({"el": 0})
+
+def test_jmespath_nested_expression():
+    expression.compile(f"a.b")
```

Comment:
"a.b" is already a string, why f-string is needed?

Reply:
fixed